packages/core/src/render3/interfaces/view.ts

```diff
@@ -49,14 +49,48 @@ export const PREORDER_HOOK_FLAGS = 18;
 /** Size of LView's header. Necessary to adjust for it when setting slots.  */
 export const HEADER_OFFSET = 20;
 
+/**
+ * Represents Angular view data.
```

Comment:
```suggestion
 * Represents Angular View.
```
The fact that it is data is internal concern not external.